client/extensions/woocommerce/state/site/status/wc-api/error-reducer.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+
+import {
+	WOOCOMMERCE_API_SET_ERROR,
```

Comment:
This error code I think will change soon with the data layer and all, so I'd like to leave it until then.